src/applications/personalization/profile/components/connected-apps/ConnectedApps.jsx

```diff
@@ -16,6 +16,7 @@ import { focusElement } from 'platform/utilities/ui';
 import Headline from '../ProfileSectionHeadline';
 import { AppDeletedAlert } from './AppDeletedAlert';
 import { ConnectedApp } from './ConnectedApp';
+import * as AlertBoxComponent from '@department-of-veterans-affairs/component-library/AlertBox';
```

Comment:
we are already importing from this module on line 7. You can update that line to be `import AlertBox, {ALERT_TYPE}...`

Reply:
yep I see it now. Will fix that